### ISSUE TITLE
@values with commas and quotes in them?

### ISSUE THREAD

**Issue by max** [salsify/ember-css-modules#22]
Is it possible to have a `@value` for a `font-family`?

```css
@value --font-family: "Helvetica Neue", Arial, Geneva, sans-serif;

.foobar {
  font-family: --font-family;
}

/* yields */

.foobar {
  font-family: "Helvetica Neue";
}
```

**Comment by dfreeman** [comment]
:sob: I'd forgotten about this. https://github.com/css-modules/postcss-modules-values/pull/23

**Comment by dfreeman** [comment]
If it comes to it, I may switch to a fork of `postcss-modules-values` until that PR moves forward, because this is a pretty obnoxious bug.

**Comment by buschtoens** [comment]
The upstream fix got merged! :)

**Comment by dfreeman** [comment]
I saw! :)
Unfortunately the loader itself is [locked to specific patch releases](https://github.com/css-modules/css-modules-loader-core/issues/119) of its constituent plugins, so this won't just start working for free.

I'll aim to cut a release with this and (hopefully) #40 in the next couple of days.

**Comment by dfreeman** [comment]
Just released v0.4.3 with a fix for this!
